addons/mail/static/src/model/model_manager.js

```diff
@@ -394,6 +415,27 @@ export class ModelManager {
     // Private
     //--------------------------------------------------------------------------
 
+    /**
+     * Returns the given data object completed with the default values of the
+     * given model.
+     *
+     * @private
+     * @param {mail.model} Model class
+     * @param {Object} [data={}]
+     */
+    _addDefaultData(Model, data = {}) {
```

Comment:
Don't like naming that's too imperative for my taste, and it's misleading. Would rename to simply `getDefaultData`.

Reply:
It doesn't get just default data, it adds them to the given data. `getDefaultData` would be misleading

Reply:
Hm okay I see, ignore comment then.